views/trust-individual-beneficial-owner.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% include "includes/list/errors.html" %}
+
+      <span class="govuk-caption-l">{{ trustData.trustName }}</span>
+      <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
+      <br> <p class="govuk-body">You can add more later.</p> <br>
+
+      <form class="form" method="post">
+
+        {% set interestedPersonDateHtml %}
+        <br>
+        {% set fieldParam = {
+          name: 'interestedPersonDate',
+          label: 'What date did they become an interested person?',
+          classes: 'govuk-fieldset__legend--s',
+          error: errors.interestedPersonDate
+        } %}
+        {% include "includes/inputs/date-input.html" %}
+        {% endset -%}
+
+        {% set fieldParam = {
+          name: 'firstName',
+          label: 'What is their first name?',
+          hint: 'This is also known as their given name.',
+          error: errors.firstName
+        } %}
+        {% include "includes/inputs/text-input.html" %}
+
+        {% set fieldParam = {
+          name: 'lastName',
+          label: 'What is their last name?',
+          hint: 'This is also known as their family name.',
+          error: errors.lastName
+        } %}
+        {% include "includes/inputs/text-input.html" %}
+
+        {% set fieldParam = {
+          name: 'dateOfBirth',
+          label: 'What is their date of birth?',
+          error: errors.dateOfBirth
+        } %}
+        {% include "includes/inputs/date-input.html" %}
+
+        {% set fieldParam = {
+          name: 'roleWithinTrust',
+          label: 'What is their role within the trust?',
+          error: errors.roleWithinTrust,
+          items: [
+            {
+              value: pageParams.trusteeType.BENEFICIARY,
```

Comment:
use `pageData` here

pageParam for whole page (title, icons, links)
pageData for form related context (like form options or drop down values)

Reply:
Updated, good explanation

Reply:
Also create new individualTrusteeType to be used here and in the controller